<ui version="4.0" stdsetdef="1" >
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow" >
    <property name="objectName" >
      <string notr="true" >MainWindow</string>
    </property>
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>812</width>
        <height>605</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Qt WebBrowser</string>
    </property>
    <widget class="QWidget" name="centralWidget">
    <layout class="QHBoxLayout" >
      <property name="objectName" >
        <string notr="true" >unnamed</string>
      </property>
      <property name="margin" >
        <number>0</number>
      </property>
      <property name="spacing" >
        <number>6</number>
      </property>
      <item>
        <widget class="QFrame" name="Frame3" >
          <property name="objectName" >
            <string notr="true" >Frame3</string>
          </property>
          <property name="frameShape" >
            <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
            <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QVBoxLayout" >
            <property name="objectName" >
              <string notr="true" >unnamed</string>
            </property>
            <property name="margin" >
              <number>1</number>
            </property>
            <property name="spacing" >
              <number>0</number>
            </property>
            <item>
              <widget class="WebAxWidget" name="WebBrowser" >
                <property name="objectName" >
                  <string notr="true" >WebBrowser</string>
                </property>
                <property name="focusPolicy" >
                  <enum>Qt::StrongFocus</enum>
                </property>
                <property name="control" >
                  <string>{8856F961-340A-11D0-A96B-00C04FD705A2}</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
    </widget>
    <widget class="QToolBar" name="tbNavigate" >
      <property name="objectName" >
        <string notr="true" >tbNavigate</string>
      </property>
      <property name="windowTitle" >
        <string>Navigation</string>
      </property>
      <addaction name="actionBack" />
      <addaction name="actionForward" />
      <addaction name="actionStop" />
      <addaction name="actionRefresh" />
      <addaction name="actionHome" />
      <addaction name="separator" />
      <addaction name="actionSearch" />
    </widget>
    <widget class="QToolBar" name="tbAddress" >
      <property name="objectName" >
        <string notr="true" >tbAddress</string>
      </property>
      <property name="windowTitle" >
        <string>Address</string>
      </property>
      <addaction name="actionGo" />
    </widget>
    <widget class="QMenuBar" name="menubar" >
      <property name="objectName" >
        <string notr="true" >menubar</string>
      </property>
      <widget class="QMenu" name="PopupMenu" >
        <property name="objectName" >
          <string notr="true" >PopupMenu</string>
        </property>
        <property name="title" >
          <string>&amp;File</string>
        </property>
        <widget class="QMenu" name="FileNewGroup_2" >
          <property name="objectName" >
            <string notr="true" >FileNewGroup_2</string>
          </property>
          <property name="title" >
            <string>New</string>
          </property>
          <addaction name="actionNewWindow" />
        </widget>
        <addaction name="FileNewGroup" />
        <addaction name="FileNewGroup_2" />
        <addaction name="separator" />
        <addaction name="actionFileClose" />
      </widget>
      <widget class="QMenu" name="BookmarksMenu" >
        <property name="objectName" >
          <string notr="true" >BookmarksMenu</string>
        </property>
        <property name="title" >
          <string>&amp;Bookmarks</string>
        </property>
        <addaction name="actionAddBookmark" />
        <addaction name="separator" />
      </widget>
      <widget class="QMenu" name="HistoryMenu" >
        <property name="objectName" >
          <string notr="true" >HistoryMenu</string>
        </property>
        <property name="title" >
          <string>Hi&amp;story</string>
        </property>
      </widget>
       <widget class="QMenu" name="unnamed" >
        <property name="objectName" >
          <string notr="true" >unnamed</string>
        </property>
        <property name="title" >
          <string>&amp;Help</string>
        </property>
        <addaction name="actionAbout" />
        <addaction name="actionAboutQt" />
      </widget>
      <addaction name="PopupMenu" />
      <addaction name="BookmarksMenu" />
      <addaction name="HistoryMenu" />
      <addaction name="unnamed" />
    </widget>
    <action name="actionGo" >
      <property name="objectName" >
        <string>actionGo</string>
      </property>
      <property name="icon" >
        <iconset>:/images/go.png</iconset>
      </property>
      <property name="iconText" >
        <string>Go</string>
      </property>
    </action>
    <action name="actionBack" >
      <property name="objectName" >
        <string>actionBack</string>
      </property>
      <property name="icon" >
        <iconset>:/images/back.png</iconset>
      </property>
      <property name="iconText" >
        <string>Back</string>
      </property>
      <property name="shortcut" >
        <string>Backspace</string>
      </property>
    </action>
    <action name="actionForward" >
      <property name="objectName" >
        <string>actionForward</string>
      </property>
      <property name="icon" >
        <iconset>:/images/forward.png</iconset>
      </property>
      <property name="iconText" >
        <string>Forward</string>
      </property>
    </action>
    <action name="actionStop" >
      <property name="objectName" >
        <string>actionStop</string>
      </property>
      <property name="icon" >
        <iconset>:/images/stop.png</iconset>
      </property>
      <property name="iconText" >
        <string>Stop</string>
      </property>
    </action>
    <action name="actionRefresh" >
      <property name="objectName" >
        <string>actionRefresh</string>
      </property>
      <property name="icon" >
        <iconset>:/images/refresh.png</iconset>
      </property>
      <property name="iconText" >
        <string>Refresh</string>
      </property>
    </action>
    <action name="actionHome" >
      <property name="objectName" >
        <string>actionHome</string>
      </property>
      <property name="icon" >
        <iconset>:/images/home.png</iconset>
      </property>
      <property name="iconText" >
        <string>Home</string>
      </property>
    </action>
    <action name="actionFileClose" >
      <property name="objectName" >
        <string>actionFileClose</string>
      </property>
      <property name="iconText" >
        <string>Close</string>
      </property>
      <property name="text" >
        <string>C&amp;lose</string>
      </property>
    </action>
    <action name="actionSearch" >
      <property name="objectName" >
        <string>actionSearch</string>
      </property>
      <property name="icon" >
        <iconset>:/images/search.png</iconset>
      </property>
      <property name="iconText" >
        <string>Search</string>
      </property>
    </action>
    <action name="actionAbout" >
      <property name="objectName" >
        <string>actionAbout</string>
      </property>
      <property name="iconText" >
        <string>About</string>
      </property>
    </action>
    <action name="actionAddBookmark" >
      <property name="objectName" >
        <string>actionAddBookmark</string>
      </property>
      <property name="text" >
        <string>Add Bookmark</string>
      </property>
    </action>
    <action name="actionAboutQt" >
      <property name="objectName" >
        <string>actionAboutQt</string>
      </property>
      <property name="iconText" >
        <string>About Qt</string>
      </property>
    </action>
    <actiongroup name="FileNewGroup" >
      <action name="actionNewWindow" >
        <property name="objectName" >
          <string>actionNewWindow</string>
        </property>
        <property name="iconText" >
          <string>Window</string>
        </property>
        <property name="shortcut" >
          <string>Ctrl+N</string>
        </property>
      </action>
      <property name="objectName" >
        <string>FileNewGroup</string>
      </property>
    </actiongroup>
  </widget>
	<customwidgets>
		<customwidget>
			<class>WebAxWidget</class>
			<extends>QAxWidget</extends>
			<header>webaxwidget.h</header>
		</customwidget>
	</customwidgets>
  <layoutdefault spacing="6" margin="11" />
</ui>
